source/educators/how-tos/course_development/enable_calculator.rst

```diff
@@ -44,5 +44,5 @@ To disable or enable the Calculator resource, follow these steps.
 +--------------+-------------------------------+----------------+---------------------------------------------------------------+
 | Review Date  | Working Group Reviewer        |   Release      |Test situation                                                 |
 +--------------+-------------------------------+----------------+---------------------------------------------------------------+
-| 07/01/2025   | Leira (Curricu.me)            | Sumac          | Fail (https://github.com/openedx/docs.openedx.org/issues/1164)|
+| 07/23/2025   | Leira (Curricu.me)            | Sumac          | Pass                                                          |
```

Comment:
@jswope00 @leiramagpantay does this mean that #1164 is no longer valid/should be closed?